cuda_bindings/cuda/bindings/nvrtc.pyx.in

```diff
@@ -85,6 +85,8 @@ class nvrtcResult(IntEnum):
     NVRTC_ERROR_PCH_CREATE = cynvrtc.nvrtcResult.NVRTC_ERROR_PCH_CREATE{{endif}}
     {{if 'NVRTC_ERROR_CANCELLED' in found_values}}
     NVRTC_ERROR_CANCELLED = cynvrtc.nvrtcResult.NVRTC_ERROR_CANCELLED{{endif}}
+
+_dict_nvrtcResult = dict(((int(v), v) for k, v in nvrtcResult.__members__.items()))
```

Comment:
two nits
1. micro optimization using `.values()` (reduce about 0.7us for creating the dict for CUresult)
2. declare it as `cdef` so that it is not accessible from Python
```suggestion
cdef _dict_nvrtcResult = dict(((int(v), v) for v in nvrtcResult.__members__.values()))
```